### ISSUE TITLE
log message: deprecated makeAllNonExistentDirectories

### ISSUE THREAD

**Issue by maphew** [leo-editor/leo-editor#1375]
When simulating a new user I noted this in the log window:
` os_path_expandExpression is deprecated. called from: create,makeAllNonExistentDirectories`

Recipe: move HOME/.leo folder where it's not found, then start Leo.

~~~
Leo Log Window
Leo 6.1-devel, devel branch, build 2a61f52701
2019-10-08 17:58:06 -0500
Python 3.6.2, PyQt version 5.12.3
Windows 10 AMD64 (build 10.0.17134) SP0
created directory: C:/Users/mattw/.leo
leoID='mhw'
.leoID.txt created in C:/Users/mattw/.leo
current dir: C:/Users/mattw
load dir: C:/users/mattw/code/leo-editor/leo/core
global config dir: C:/users/mattw/code/leo-editor/leo/config
home dir: C:/Users/mattw
reading settings in C:/users/mattw/code/leo-editor/leo/config/leoSettings.leo
Using default leo file name:
C:/Users/mattw/~/.leo/workbook.leo
 os_path_expandExpression is deprecated. called from: create,makeAllNonExistentDirectories
created: C:/Users/mattw/.leo/spellpyx.txt
reading settings in C:/users/mattw/code/leo-editor/leo/doc/CheatSheet.leo
~~~

**Comment by edreamleo** [comment]
@maphew Strange.  If anything, I intended to deprecate g.makeAllNonExistentDirectories.

I'll see if I can clean up this mess.  A deprecation message does not seem advisable.

**Comment by edreamleo** [comment]
Fixed at 480f9fa in devel.

g.os_path_expandExpression(s, c=c) has been deprecated in favor of c.os_path_expandExpression(s).

g.makeAllNonExistentDirectories contained the only call to g.os_path_expandExpression.
